### PR TITLE
[e2e] Clean up e2e/framework/upgrade_util.go

### PR DESCRIPTION
**Pull request by jiatongw**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Reference issue #76206 (parent issue #75601 )

**Special notes for your reviewer**:

Considering move this file to `framework/cluster` package

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by SataQiu** [comment]
Thanks @jiatongw 
/release-note-none
/priority backlog
/ok-to-test
/cc @SataQiu

File: test/e2e/framework/upgrade_util.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func RealVersion(s string) (string, error) {
 	Logf("Getting real version for %q", s)
 	v, _, err := RunCmd(path.Join(TestContext.RepoRoot, "hack/get-build.sh"), "-v", s)
 	if err != nil {
-		return v, err
+		return v, fmt.Errorf("getting real version for %q error: %v", s, err)
```
Comment (andrewsykim) [inline test/e2e/framework/upgrade_util.go]: nit: "error getting"

**Comment by andrewsykim** [comment]
> Considering move this file to framework/cluster package

Thoughts on  `test/e2e/framework/version` or `test/e2e/framework/clusterversion`?

**Review by fejta — APPROVED** [review]
/lgtm
/approve

**Comment by jiatongw** [comment]
> > Considering move this file to framework/cluster package
> 
> Thoughts on `test/e2e/framework/version` or `test/e2e/framework/clusterversion`?

I am thinking that maybe there are more cluster related test framework will be added in the future. So I prefer to put it in a more generic package. Next time new code can be also added here. What do you think? :smiley:

File: test/e2e/framework/upgrade_util.go (hunk 1 above)
Comment (jiatongw) [reply test/e2e/framework/upgrade_util.go]: Thanks, that's better. Fixed

**Comment by andrewsykim** [comment]
sgtm!

**Review by fejta — COMMENTED** [review]
/lgtm

**Pull request merged**